indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -1538,6 +1541,20 @@ public Response setEndOffsetsHTTP(
     return setEndOffsets(sequences, finish);
   }
 
+  @POST
+  @Path("updatePendingSegmentMapping")
```

Comment:
```suggestion
  @Path("pendingSegmentMapping")
```
POST method implies update

Reply:
done